app/controllers/V2App.scala

```diff
@@ -39,6 +40,12 @@ class V2App(isDev: Boolean, val acl: Acl, dynamo: Dynamo, val deps: BaseFaciaCon
     val maybeUserData: Option[UserData] = Scanamo.exec(dynamo.client)(
       userDataTable.get('email -> userEmail)).flatMap(_.right.toOption)
 
+    val clipboardArticles = if (editingEdition)
```

Comment:
As discussed, this is fine for now, but it'd be brilliant to have our data model take care of this logic by itself with a Map[EditingMode, ClipboardArticles] or similar.

Reply:
I agree but I think that could be part of a larger change to move away from an editing editions boolean to a proper editing mode flag. We're currently littering the code base with `isEditingEditions` and I'd prefer this was a enum such as `EditingMode`

Reply:
absolutely 👍